AOJ/03/03_C.py

```diff
@@ -0,0 +1,8 @@
+while True:
+    x,y = map(int, input().split())
+    if x>y:
+        print('{} {}'.format(y,x))
```

Comment:
03_Bではf-stringを使い `print` 文を書いていますので、書き方を統一するようにしてください。
```suggestion
        print(f'{x} {y}')
```